src/Account.java

```diff
@@ -30,6 +31,18 @@ public Account(Profile profileHolder, double balance){
      */
     public abstract double monthlyFee();
 
+    // get Account Type
+    public abstract String getAccountType();
+    // Abstract method to get the profile of the account
+    public abstract Profile getProfileType();
+
+    public abstract double getBalance();
```

Comment:
why is getBalance abstract?  Every implementation of it in a subclass is identical no?